frontend/src/components/SelectPagination.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Pagination from 'react-js-pagination';
+import { Dropdown } from '@trussworks/react-uswds';
+
+export function renderTotal(title, offset, perPage, activePage, count) {
+  const from = offset >= count ? 0 : offset + 1;
+  const offsetTo = perPage * activePage;
+  let to;
+  if (offsetTo > count) {
+    to = count;
+  } else {
+    to = offsetTo;
+  }
+  return `Showing ${from}-${to} of ${count} ${title.toLowerCase()}`;
+}
+
+export const SelectPagination = ({
```

Comment:
no need to export twice (you are exporting as default below, which is our normal practice)